sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java

```diff
@@ -49,24 +49,24 @@
     private boolean decommissioned;
 
     /*
-     * The model definition.
+     * The model definition that conforms to the Digital Twins Definition Language (DTDL) v2.
```

Comment:
Can you add a link to the documentation, like I did in the .NET PR? Check your work item for a link.

Reply:
I will take care of this in next PR.